### PR TITLE
fix(hmi-client): dataset upload

### LINKED ISSUE #1196: [BUG]: Upload button on modal for 'Upload resources' feature doesn't work
**Describe the bug**
Here I try to upload a csv and it doesn't work.

**To Reproduce**

https://github.com/DARPA-ASKEM/Terarium/assets/28237258/239caffe-f93e-4a50-8b0a-822194d5af10

### PR DESCRIPTION
**Pull request by dvince2**
A few fixes here:
* Secret file commit is intentional here. There was still something wrong with the last try
* Better notification of errors with toast.
* After upload from disk project is refereshed and dialog closed
* Github works better but still a bit of a WIP

Resolves #1196

### CODE REVIEW

File: packages/client/hmi-client/src/components/extracting/tera-drag-n-drop-file-previewer.vue [hunk 1]
```diff
@@ -59,32 +59,20 @@ const props = defineProps({
...
 	() => props.isProcessing,
-	() => {
-		if (props.isProcessing) {
-			progress.value = 0;
-			progressUpdater = setInterval(updateProgress, 50);
-		} else {
-			clearInterval(progressUpdater);
-		}
-	}
+	() => props.progress
```
Comment (shawnyama) [inline packages/client/hmi-client/src/components/extracting/tera-drag-n-drop-file-previewer.vue:75]: I don't think this watcher does anything.

File: packages/client/hmi-client/src/page/project/components/tera-project-overview.vue [hunk 2]
```diff
@@ -228,7 +230,9 @@ const resources = useResourcesStore();
 const isEditingProject = ref(false);
 const inputElement = ref<HTMLInputElement | null>(null);
 const newProjectName = ref<string>('');
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const visible: Ref<boolean> = ref(false);
```
Comment (shawnyama) [inline packages/client/hmi-client/src/page/project/components/tera-project-overview.vue:234]: Unused - causing the lint error

Comment (dvince2) [reply packages/client/hmi-client/src/page/project/components/tera-project-overview.vue:234]: No you're right... thats causing the lint error, and our linter is choosing to ignore the `disable` above.... hmmm

Comment (dvince2) [reply packages/client/hmi-client/src/page/project/components/tera-project-overview.vue:234]: `visible` _is_ being used though at the bottom of the file.

**Review by Tom-Szendrey — APPROVED** [review]

**Pull request merged**